js/src/options.ts

```diff
@@ -20,6 +20,7 @@ export type Options = {
     onPreferencesChanged?: (value: string) => any;  // Executes when the user's preferences have changed.
     disableGrammar?: boolean;       // Option to disable Grammar
     disableTranslation?: boolean;   // Option to disable translation
+    disableLanguageDetection?: boolean;      // SDK option override to disable lang detection and use only the explicitly tagged language
```

Comment:
Can you please use the text I provided earlier, about being tagged on the chunk? I think that is a bit more clear here in this code in case customers are looking at this and trying to understand what we mean.

Also, spell out Language Detection and capitalize as a feature name, better/more formal to not have 'lang' abbreviation

Reply:
done